.github/workflows/generator_container_slsa3.yml

```diff
@@ -70,6 +70,11 @@ on:
         required: false
         type: string
         default: ""
+      cosign-repository:
```

Comment:
Let's rename this to a neutral (non-cosign). Nothing against cosign (we use it!) but it's an implementation detail which need not be exposed to users, I think.
How about `attestation-registry`? 

Also, update the text accordingly: `pushed to this registry instead of image registry`

Does that make sense?

Reply:
I like the name `attestation-store` but I think most of our workflows use `provenance` for the terminology in inputs etc. so maybe `provenance-store` is better?

Reply:
provenance is fine with me, good call. I'm thinking that probably `provenance-registry` is better than `provenance-store`, mostly because there's no standard interface to "talk" to an attestation store. If someone wanted to support storing attestations in a GCP bucket, we'd have to parse the `provenance-store` value to identify which kind of store it is. Using `provenance-registry` clearly indicates that "kind" of registry. If we want to support other "kinds" of store, we would add `provenance-bucket`, `provenance-xxx`. Or we keep `provenance-store` but require a scheme in the uri, like "container://docker.io/xxx". Another option is to have a seperate `store-type`.

Wdut?

Reply:
SGTM. `provenance-registry` it is then.

Reply:
resorted to use `provenance-registry`